session_items.py

```diff
@@ -43,12 +46,13 @@ def add_item(title):
     items = get_items()
 
     # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
+    id = len(items) + 1 if items else 0
+    #print(len(items))
     item = { 'id': id, 'title': title, 'status': 'Not Started' }
 
     # Add the item to the list
     items.append(item)
+    items= sorted(items,key=lambda d: (d['status'],d['title']))
```

Comment:
You use this sorting lambda in a couple of places and you want to ensure they are kept the same. So it would be good to extract it into a function that you can re-use. 

Also, why is its parameter called `d`?